src/main/java/neqsim/processSimulation/processEquipment/pump/Pump.java

```diff
@@ -329,82 +327,6 @@ public void setPressure(double pressure, String unit) {
         pressureUnit = unit;
     }
 
-    public static void main(String[] args) {
```

Comment:
Kunne tenkt meg å beholde disse i main metoder (for tester av funksjonalitet i filer). MEn ser at det blir mye main metoder - som kunne vært flyttet.